crates/host/src/wasmbus/mod.rs

```diff
@@ -1604,15 +1649,16 @@ fn linkdef_hash(
     hex::encode_upper(hash.finalize())
 }
 
-#[instrument(level = "debug", skip_all)]
-async fn create_lattice_metadata_bucket(
+#[instrument(skip(jetstream, bucket_message_name))]
+async fn create_bucket(
     jetstream: &async_nats::jetstream::Context,
     bucket: &str,
-) -> anyhow::Result<()> {
+    bucket_message_name: &str,
```

Comment:
Nit: is this needed? I think I'd prefer to see the actual name for debug purposes